cfgov/core/utils.py

```diff
@@ -67,11 +69,10 @@ def format_file_size(bytecount, suffix='B'):
     return "{:.0f} {}{}".format(bytecount, 'T', suffix)
 
 
-def get_link_tags(soup):
+def get_link_tags(html):
     tags = []
-    for a in soup.find_all('a', href=True):
-        if not is_image_tag(a):
-            tags.append(a)
+    for a in A_TAG.findall(html):
+        tags.append(a)
```

Comment:
Now that the image-related logic has been removed, could this be simplified into a one-liner?

```py
return A_TAG.findall(html)
```

Reply:
Yes, it could!